app/assets/stylesheets/components/_phase-banner.scss

```diff
@@ -0,0 +1,10 @@
+.gem-c-phase-banner {
```

Comment:
As in the other pull request, this is an overwrite should stay in `app/assets/stylesheets/utilities/_overwrites.scss`